main.tf

```diff
@@ -0,0 +1,45 @@
+data "template_file" "body" {
+  template = "${var.body}"
+
+  vars = "${var.vars}"
+}
+
+data "template_file" "subject" {
+  template = "${var.subject}"
+
+  vars = "${var.vars}"
+}
+
+locals {
+  body    = "${data.template_file.body.rendered}"
+  subject = "${data.template_file.subject.rendered}"
+  command = "${var.mail_command} ${join(" ", var.to)}"
+}
+
+resource "null_resource" "default" {
```

Comment:
why do we need the "null_resource"?
(not used anywhere)

Reply:
It's used to run the `local-exec` provisioner. Is there another kind of resource we can use?